code/__HELPERS/unsorted.dm

```diff
@@ -582,7 +582,7 @@ Takes: Area type as text string or as typepath OR an instance of the area.
 
 Returns: A list of all areas of that type in the world.
 */
-/proc/get_areas(areatype, subtypes=TRUE)
+/proc/get_areas(areatype, target_z = 0, subtypes=TRUE)
```

Comment:
You checked every usage of this right? Adding an arg in the middle is a bold move.

Reply:
Also why are you not using null? Isn't 0 the station z?

Reply:
2 is station z, I've copypasted it from one proc down which uses same logic. It could be changed to work with null.
And yes, I've replaced every usage of this proc.